css/main.css

```diff
@@ -458,6 +461,100 @@ body {
   margin: 18px 0;
 }
 
+.modal {
+  background: #fff;
+  padding: 1.5rem;
+  width: 90%;
+  opacity: 0;
```

Comment:
- [ ] Please make sure the project details and project technologies are aligned on the same axis horizontally. The project description should be on the left side and project technologies and buttons should be on the right side.

### Your design

![image](https://user-images.githubusercontent.com/83803414/167876143-6b9098e3-a719-41e5-b255-b08f8de7da92.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/167876192-ee572128-8a86-4611-beef-2cc9b7cdd7bb.png)